start.go

```diff
@@ -46,9 +46,9 @@ func start(c *cli.Context) error {
 		"-p", "8080:8080",
 	}
 	if c.String("log-level") != "" {
-		args = append(args, "-e", fmt.Sprintf("GIN_MODE=%v", c.String("log-level")))
+		args = append(args, "-e", fmt.Sprintf("FN_LOG_LEVEL=%v", c.String("log-level")))
 	}
-	if c.String("env-file") != ""{
```

Comment:
OT: @denismakogon looks like you're not running `go fmt`.

Reply:
we should put that into CI, but yeah, have fmt checks for fn repo in pre-commit hooks.

Reply:
Regardless of CI, it looks like your fmt isn't working.